samples/snippets/visualbasic/VS_Snippets_Wpf/WindowIconSnippets/visualbasic/windowiconsample.vbproj

```diff
@@ -21,7 +21,7 @@
     <IsWebBootstrapper>true</IsWebBootstrapper>
     <BootstrapperEnabled>true</BootstrapperEnabled>
     <PublishUrl>Publish\</PublishUrl>
-    <ApplicationIcon>SytemIcon.ico</ApplicationIcon>
+    <ApplicationIcon>SystemIcon.ico</ApplicationIcon>

```

Comment:
Is there a file this refers to that needs to be renamed as well?

Reply:
Good catch! yes!